index.d.ts

```diff
@@ -356,7 +356,9 @@ declare module 'regular-table' {
     global {
         namespace JSX {
             interface IntrinsicElements {
-                "regular-table": DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>;
+                "regular-table": Omit<DetailedHTMLProps<HTMLAttributes<HTMLElement>, HTMLElement>, "className"> & {
```

Comment:
What's the minimum supported TypeScript version? `Omit` was [introduced in 3.5](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-5.html#the-omit-helper-type), but I can also define a userland version of `Omit` if there are concerns.

Reply:
Per the issue description which links to https://reactjs.org/docs/web-components.html#using-web-components-in-react

```js
<regular-table className="some-class" />
```

doesn't work and you need to do

```js
<regular-table class="some-class" />
```

but the current typings don't allow that. With my change this becomes possible.